speechbrain/lobes/models/discrete/speechTokenizer.py

```diff
@@ -61,6 +58,18 @@ def __init__(
         source,
         save_path,
     ):
+
+        # Lazy import to avoid circular dependency issues
+        try:
```

Comment:
same as the above comments.
Regarding implementation on SpeechBrain, it should be feasible, but it would require porting a significant amount of code from the original repository. We need to establish a clear policy concerning audio tokenizers. This is a rapidly evolving field, with new tokenizers introduced almost monthly. While implementing all from scratch could make our code cleaner and less dependent, it might hinder our ability to keep pace with the field. Moreover, many of them introducing a new version, we need to make sure we are updated with their changes.

Reply:
Well the entire field is evolving pretty rapidly, which is not preventing us from implementing things in SB. For SSL models, we got lucky enough that HuggingFace decided to do the transformers library. I'll write a comment on the Slack about an idea I just got for this problem.